transcom/mymove
# INT B-18629 Add Origin GBLOC To Move Screen (Backend)

## [B-18629](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-18629)

## Summary

This is to add Origin Duty Location GBLOC alongside the rest of the move header information (origin and destination information. This information should be visible to all Government users regardless of the page that they are on (SC, TOO, TIO and QAE/CSR).

This is only the Backend portion of the code and I have added OriginDutyLocationGBLOC to show on the returned payload data when retrieving `Order` using `GET`. GET request for `Order` is being called in all of the areas mentioned above - so this will be used in the Frontend implementation of the code.

### How to test

1. Create a move.
2. As a SC, TOO, TIO or QAE/CSR -> Open up your browser Developer's Tool(Inspect tool) and open a move.
3. Use a network tab to search for `GET` request for `Order`(shown in image below).
4. Make sure you see `originDutyLocationGBLOC` in the preview tab and it matches the one you saw in the move queue table(where you search for moves).
5. See the images below for reference.

## Screenshots
![image](https://github.com/transcom/mymove/assets/146856854/9dfa34eb-be68-4f8b-bf99-0248bd3cb918)

![image](https://github.com/transcom/mymove/assets/146856854/4d79fe0f-a267-45fc-8a70-4b4428108113)
![image](https://github.com/transcom/mymove/assets/146856854/6d79f339-f393-4002-9294-b57503b8ef4d)
